src/helpers/distributor.js

```diff
@@ -194,11 +201,29 @@ class Distributor {
         break;
 
       default:
-        outputs.forEach(it => logger.raw(`[${it.component}] ${(Buffer.from(it.buffer).toString('utf8'))}`));
+        // @todo: investigate if this row is really required
+        results.forEach(it => logger.raw(`[${it.component}] ${(Buffer.from(it.buffer).toString('utf8'))}`));
         break;
     }
   }
 
+  /**
+   * @param {Object[]} results
+   * @private
+   */
+  _handleWorkspaceList(results) {
+    const result = {};
+
+    results.forEach(item => {
```

Comment:
I guess it can be simplified like this
```javascript
const result = results.reduce((acc, item) => {
  item.workspaces.filter(it => !acc[it]).forEach(it => acc[it] = {});
  acc[item.activeWorkspace][item.component] = null;

  return acc;
}, {});
```